docs/_assets/codepen.js

```diff
@@ -102,6 +102,15 @@ CodeBlockCodePen.prototype.clickHandler = function(form, pre) {
 
   return function() {
 
+    // Track codepen button clicks
+    if (typeof ga !== 'undefined') {
+      ga('send', {
+        hitType: 'event',
+        eventCategory: 'codepen',
+        eventAction: 'click'
```

Comment:
The metric is pretty useless like that, isn’t it?. How about you give the `<pre>`s a `data-name` attribute or something that you can query for the event?